composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "a2lix/translation-form-bundle":      "^2.1",
-        "egeloen/ckeditor-bundle":            "^4.0",
+        "friendsofsymfony/ckeditor-bundle": "dev-master",
```

Comment:
I didn't change this to "^1.0" do you want me to open a new PR or you will do it?

Reply:
cc @semin-lev  @IgorNikolaev

Reply:
We'll do it, thank you.

Reply:
@kunicmarko20 if it's not hard for you, please, open new PR. Thank you.

Reply:
@kunicmarko20 we did, thank you.